src/resources/cdm-detailed.properties

```diff
@@ -102,16 +102,16 @@ spark.cdm.connect.target.password      cassandra
 #-----------------------------------------------------------------------------------------------------------
 spark.cdm.schema.origin.keyspaceTable                keyspace_name.table_name
 
-# Max TTL value of all non-PK columns will be used for insert on target
+# Max TTL value of all TTL-eligible non-PK columns will be used for insert on target
 #spark.cdm.schema.origin.column.ttl.automatic         true
 
-# Max TTL value of specified non-PK columns will be used for insert on target (overrides automatic setting)
```

Comment:
Curious, why are we removing `non-PK` verbiage from here as there won't be any writetime/ttl for primary key columns of a C* table?

Reply:
The specified columns are implicitly non-PK columns - if they are not, validation will fail and feature will be disabled :)

Reply:
I can add it back in (in parentheses)

Reply:
I think we should add it back as only non-PK columns has the writimetime/ttl values/properties.

Reply:
@arvy @arvydasj could you pls take care of this trivial doc update pr? Thank you in advance!